src/pages/Trade/components/EtherPromoCard/EtherPromoCard.js

```diff
@@ -0,0 +1,107 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { useTranslation, Trans } from 'react-i18next';
+import styled from 'styled-components';
+
+import { getWalletInfo } from '../../../../ducks';
+
+import history from '../../../../utils/history';
+
+import { ROUTES } from '../../../../constants/routes';
+import { LINKS } from '../../../../constants/links';
+
+import { ExternalLink, CurrencyKey } from '../../../../shared/commonStyles';
+
+import Card from '../../../../components/Card';
+import { ButtonPrimary } from '../../../../components/Button';
+import { SYNTHS_MAP, CRYPTO_CURRENCY_MAP } from '../../../../constants/currency';
+import { getCurrencyKeyBalance } from '../../../../utils/balances';
+
+const ETH_CURRENCY_KEY = CRYPTO_CURRENCY_MAP.ETH;
```

Comment:
it reminds me I need to expose the synths in `synthetix-js` without calling `setContractSettings()` first